woocommerce/woocommerce-android
# Add fee view for Order Creation feature

Summary
==========
Fixes issue #5695 by introducing the Add fees view, this view differs significantly from the designs since we only have the iOS version that suggests the usage of a toggle that we don't adopt on WCAndroid. The UI decision went to the simplest possible solution with a switch and leave to discuss later how to better introduce the percentage toggle.

Screenshots
==========
| Add fee inside creation form | Fee Amount | Fee Percentage |
| ------------- | ------------- | ------------- |
| ![Screenshot_20220215_004723](https://user-images.githubusercontent.com/5920403/153989747-24a44cca-efbd-406a-8670-e8eacc0b9190.png) | ![Screenshot_20220215_004730](https://user-images.githubusercontent.com/5920403/153989807-d9066457-ebc3-4c2b-bd7b-9f11e089ab86.png) | ![Screenshot_20220215_004737](https://user-images.githubusercontent.com/5920403/153989820-22e0671e-be24-4424-ab9b-44f555c3f439.png) |

How to Test
==========
1. Make sure the feature flag `ORDER_CREATION_M2` is enabled.
2. Open order creation form.
3. Add a product.
4. Notice the button "Add fee" on the payment section and click on it.
5. Enter an amount and a name.
6. Notice that `onNewFeeSubmitted` is called with the correct data, the Order Creation flow returns to the main form but no changes have been submitted to the draft.


Update release notes:
